### DISCUSSION TITLE
Does MJML have it's own fonts?

### DISCUSSION THREAD

**Discussion by Cwasiniak** [mjmlio/mjml#2862] [Q&A]
I cant figure out what syntax to use when adding fonts, embedded or downloaded. Does MJML host it's own fonts? or, have a set of fonts that can be used?

**Comment by iRyusa** [comment]
Just check https://documentation.mjml.io/#mj-font
And few fonts are available when used in a documents https://github.com/mjmlio/mjml/blob/master/packages/mjml-core/src/index.js#L102-L110
